build/parser.js

```diff
@@ -465,13 +465,14 @@ class Parser {
    * Limit items to tradable/untradable if specified.
    */
   addTradable (item) {
-    const tradableTypes = ['Upgrades', 'Fish', 'Key', 'Focus Lens', 'Relic', 'Rifle Mod',
+    const tradableTypes = ['Arcane', 'Fish', 'Focus Lens', 'Relic', 'Rifle Mod',
```

Comment:
why'd you remove Upgrades?

Reply:
I noticed that it wasn't doing anthing, but I can have it back if you want.
Removing it didn't change the JSON Data when I ran the `npm run build`

Reply:
Can confirm after further testing that I got the exact same results with and without 'upgrades'

Reply:
okiedoke. feel free to leave it out.